depmapomics/copynumbers.py

```diff
@@ -144,15 +144,13 @@ def managingDuplicates(samples, failed, datatype, tracker, newname="arxspan_id")
 
 def pureCNpostprocess(
     refworkspace,
-    setEntity="sample_set",
     sortby=[SAMPLEID, "Chromosome", "Start", "End"],
     todrop=[],
     lohcolname=PURECN_LOH_COLNAME,
     failedcolname=PURECN_FAILED_COLNAME,
-    sampleset="all",
+    sampleset=PURECN_SAMPLESET,
```

Comment:
why can't it work on "all"? even if some failed..

Reply:
Because PureCN segs are aggregated on terra and there has to be a sampleset on terra for all the lines we have PureCN for